mail_footer_notified_partners/models/mail_followers.py

```diff
@@ -48,16 +48,17 @@ def get_signature_footer(
 
         return footer
 
-    @api.model
+    @api.multi
     def _notify(
-            self, message_id, partners_to_notify=None, force_send=False,
+            self, partners_to_notify=None, force_send=False,
             user_signature=True):
+        self.ensure_one()
         ctx = self.env.context.copy()
         if not self.env.context.get('mail_notify_noemail'):
             ctx.update({
                 'partners_to_notify': partners_to_notify,
             })
-        return super(MailNotification, self)._notify(
-            message_id,
+        return super(MailNotification, self._model)._notify(
```

Comment:
It was just a revert to the last working state... just to make it work, since a client of mine stressed about the missing partners in footer since the last update of the module - with api.model it simply didn't work.